ci/docs-builder.py

```diff
@@ -32,12 +32,21 @@ def load_config(config_name):
     config_file = os.path.join(os.path.dirname(__file__),
                                'config', 'releases', '%s.yaml' % config_name)
     if not os.path.exists(config_file):
-        print "Error: %s not found. " % config_file
+        print("Error: %s not found. " % config_file)
         sys.exit(1)
     with open(config_file, 'r') as config_handle:
         config = yaml.safe_load(config_handle)
     return config
 
+
+def update_version(config_name):
+    script_dir = os.path.abspath(os.path.dirname(__file__))
+    # script does not push to github, but it does check out all the repos we want and
+    # makes sure their version reflect the versions in the build config being used
+    script = os.path.join(script_dir, 'update-version-and-merge-forward.py')
```

Comment:
What do you think about making the `update-version-and-merge-forward` import without side effects. All we need to do is to put all the module level code in a method like `main()` and then add an `if __name__ == "__main__":` that would call `main()`. Then we could import that from here and call `main()` ourselves.